crates/light-client-prover/src/da_block_handler.rs

```diff
@@ -153,7 +159,7 @@ where
         // Do any kind of ordering etc. on batch proofs here
         // If you do so, don't forget to do the same inside zk
         let batch_proof_method_id = self
-            .batch_proof_code_commitments_by_spec
+            .batch_proof_code_commitments
             .get(&SpecId::Genesis)
             .expect("Batch proof code commitment not found");
```

Comment:
we'll need to get method id by batch_proof_output.last_l2_height's spec id

Reply:
Done